res/controllers/Pioneer-DDJ-200-scripts.js

```diff
@@ -36,6 +48,15 @@ DDJ200.init = function() {
     }, true);
 };
 
+DDJ200.listemHotcues = function(vgroup) {
```

Comment:
"listen"

Also, this method is only called in a single location, so I don't think it's needed at all and can just be I lined there.

Reply:
I was trying just to make the code cleaner,  now is inside init

Reply:
What is _listemHotcues_ supposed to mean?

Reply:
This was to listen Hotcues changes from mixxx interface and apply to the controller, but already removed the method.